src/main/java/picard/util/BedToIntervalList.java

```diff
@@ -64,9 +64,16 @@
             "     -Strand - Indicates +/- strand for the interval (either + or -) <br /> " +
             "     -Interval name - (Each interval should have a unique name) " +
             "</pre>" +
-            "  <br /><br />" +
-            "This tool requires sequence dictionary file (with \".dict\" extension), which can be created from a reference sequence " +
-            "using Picard's CreateSequenceDictionary tool."+
+            "<br/>" +
+            "This tool requires sequence dictionary, provided with the SEQUENCE_DICTIONARY or SD argument. " +
+            "This argument can be any of:" +
+            "<pre>" +
+            "    - A file with .dict extension generated using Picard's CreateSequenceDictionaryTool</br>" +
+            "    - A <name>.fa or <name>.fasta file with a <name>.dict in the same directory</br>" +
+            "    - Another IntervalList that contains the dictionary to use</br>" +
+            "    - A VCF that contains #contig lines from which to generate a sequence dictionary</br>" +
+            "    - A SAM or BAM file with @SQ lines in the header from which to create a dictionary</br>" +
```

Comment:
Not currently by the look of it.  It detects the file type by extension and CRAM isn't there currently.